dash/webapp/package.json

```diff
@@ -6,7 +6,6 @@
     "dev": "next",
     "build": "next build",
     "export": "next export",
-    "write-report": "node write-report.js",
```

Comment:
Why is this being removed?

Reply:
Oops, this is an unrelated change that snuck in – `write-report.js` no longer exists, so this command doesn't work, and I must have discovered it while working on this PR.